perllib/FixMyStreet/Template.pm

```diff
@@ -159,8 +159,11 @@ sub html_paragraph_email_factory : FilterFactory('html_para_email') {
 sub sanitize {
     my $text = shift;
 
+    # In case of markdown variant style of <https://www.google.com>
+    $text =~ s/<(\s?)+(https?.(?#>)*?)(\s?)+>/$2/g;
```

Comment:
`(\s?)+` should be `\s*`. I'm not sure what `(?#...)` is commenting here?
I think something like `s/<\s*(https?[^\s>]?)\s*>/$1/g` would be a bit more straightforward; this current one will turn `< https://www and then some <b>bold</b> text` into `https://www and then some <bbold</b> text`

Reply:
`s/<\s*(https?[^\s>]?.*?)\s*>/$1/g;` Would that be ok or is that negating what you've added as doesn't strip the chevrons as is...

Reply:
Oh, sorry, my final `?` should have been a `+` - you don't ever want any > or whitespace after the http, do you, so you don't want to use `.`